.github/workflows/trigger-all.yml

```diff
@@ -0,0 +1,40 @@
+name: build and test jobs for nightly and release
+run-name: ${{ github.actor }} triggered nightly on ${{ github.ref }}
```

Comment:
Nit: "triggered nightly" should probably be "triggered all"?

Reply:
thanks for catching it, updated.